### PR TITLE
docs(consolidation): add migration notice

### PR DESCRIPTION
**Pull request by fuxingloh**
#### What this PR does / why we need it:

Add deprecated and migration notice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 [![codecov](https://codecov.io/gh/JellyfishSDK/whale/branch/main/graph/badge.svg?token=kBCC9qSRrA)](https://codecov.io/gh/JellyfishSDK/whale)
 [![npm](https://img.shields.io/npm/v/@defichain/whale-api-client)](https://www.npmjs.com/package/@defichain/whale-api-client)
 
+## Following the migration path outline in [jellyfish#580](https://github.com/JellyfishSDK/jellyfish/issues/580), this repository being consolidated into [JellyfishSDK/jellyfish/apps/whale-api](https://github.com/JellyfishSDK/jellyfish/tree/main/apps/whale-api).
```
Comment (eli-lim) [inline README.md:5]:
```suggestion
## Following the migration path outlined in [jellyfish#580](https://github.com/JellyfishSDK/jellyfish/issues/580), this repository will be consolidated into [JellyfishSDK/jellyfish/apps/whale-api](https://github.com/JellyfishSDK/jellyfish/tree/main/apps/whale-api).
```

File: README.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
...
+
+The docker artifact will be renamed
+to [ghcr.io/jellyfishsdk/whale-api](https://github.com/JellyfishSDK/jellyfish/pkgs/container/whale-api). The npm
+packages published versions will be bumped to follow jellyfish release cycle.
```
Comment (eli-lim) [inline README.md:9]:
```suggestion
to [ghcr.io/jellyfishsdk/whale-api](https://github.com/JellyfishSDK/jellyfish/pkgs/container/whale-api). The versions
of the published npm packages will be bumped to follow jellyfish's release cycle.
```

**Review by eli-lim — COMMENTED** [review]
😛 :heart:

**Comment by fuxingloh** [comment]
> 😛 ❤️

Thanks 🫂

**Pull request merged**